config/config.ts

```diff
@@ -0,0 +1,15 @@
+import * as fs from 'fs'
+
+let configFile
+let localConfig
+
+// If project is being run locally, the developer will have their API key in the json file
+try {
+  configFile = fs.readFileSync('./config/hollowbot.config.json', 'utf8')
+  localConfig = JSON.parse(configFile)
```

Comment:
Regardless of why this gets thrown, The final error would come from the `activate.ts` which would be along the lines of  `Invalid Token for Discord API..` I didn't cover all the possibilities with this error because the main assumption is that the developer has a token to develop the bot and either he has injected it into the environment or to his config file. (I'd explain it on the README with an example JSON.)